spec/controllers/internal_api/v1/pillars_controller_spec.rb

```diff
@@ -13,6 +15,7 @@
   describe "GET /pillar" do
     before do
       Pillar.create pillar: "dashboard", value: "dashboard.example.com"
+      DockerRegistry.create url: "https://example.com", certificate: certificate
     end
 
```

Comment:
It would be nice to check some content here instead of just http status, no?

Reply:
Hmm, I think I'm not following :/

Reply:
Oh, that we expand that test to also check the contents of the response! Yeah...

Reply:
Exactly!

Reply:
Done :)